Changelogs for Digital Twin Manager Plugins 2.1.4 (#3399)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-5-0-Updated-sub-assets-and-asset-tree-routes-to-use-group-context.md

---
date: ''

> title: Updated sub-assets and asset tree routes to use '/group/:id' context

Reviewer: ```suggestion
title: Updated subassets and asset tree routes to use the /group/:id context
```

> title: Updated subassets and asset tree routes to use the /group/:id context

Reviewer: ```suggestion
title: Updated subassets and asset tree routes to avoid context issues
```

product_area: Application enablement & solutions
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-Tl88RYb4A
    label: Digital Twin Manager
build_artifact:
  - value: tc-wYIY0MBDO
    label: dtm
ticket: CTM-1997
version: 1021.4.0
---

> The subassets and asset tree contexts has been changed from `/asset/:id` to `/group/:id` to align with DTM’s registered context. The previous `asset/:id` context was not recognized globally, leading to issues such as broken tab switching - especially when external plugins were integrated. Switching to the `group/:id` context ensures reliable routing, consistent context handling, and seamless plugin compatibility across the platform.

Reviewer: ```suggestion
The subassets and asset tree contexts have been changed from `/asset/:id` to `/group/:id` to align with the Digital Twin Manager´s registered context. The previous `asset/:id` context was not recognized globally, leading to issues such as broken tab switching, especially when external plugins were integrated. Switching to the `group/:id` context ensures reliable routing, consistent context handling, and seamless plugin compatibility across the platform.
```

> Refactored route paths from /asset/:id to /group/:id for subassets and related tabs to align with DTM’s registered context. The previous asset/:id context was not recognized globally, leading to issues such as action dispatch errors and broken tab switching—especially when external plugins were integrated. Switching to the existing group/:id context ensures reliable routing, consistent context handling, and seamless plugin compatibility across the platform.

Reviewer 2: @BeateRixen , this is the content we want to convey but couldn't get it customer friendly. Please help us refine.

> Refactored route paths from /asset/:id to /group/:id for subassets and related tabs to align with DTM’s registered context. The previous asset/:id context was not recognized globally, leading to issues such as action dispatch errors and broken tab switching—especially when external plugins were integrated. Switching to the existing group/:id context ensures reliable routing, consistent context handling, and seamless plugin compatibility across the platform.

Reviewer: ```suggestion
The path for accessing subassets and related tabs has been changed from `/asset/:id` to `/group/:id` to align with DTM’s registered context. The previous `asset/:id` context was not recognized globally, leading to issues such as action dispatch errors and broken tab switching - especially when external plugins were integrated. Switching to the `group/:id` context ensures reliable routing, consistent context handling, and seamless plugin compatibility across the platform.
```
